### PR TITLE
region: update region-size from 96MB to 256MB.

### PR DESCRIPTION
**Pull request by LykxSassinator**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Ref: https://github.com/tikv/tikv/issues/17309

This pr specify the newly updated configuration change to the default value of `region-size`, changing it from `96MB` to `256MB`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: best-practices/massive-regions-best-practices.md [hunk 1]
```diff
@@ -134,7 +134,7 @@ config set merge-schedule-limit 8
 
 ### 方法六：调整 Region 大小
 
-Region 默认的大小约为 96 MiB，将其调大也可以减少 Region 个数，具体介绍可参考[使用更大的 Region](/tune-region-performance.md)。
+Region 默认的大小约为 256 MiB，将其调大也可以减少 Region 个数，具体介绍可参考[使用更大的 Region](/tune-region-performance.md)。
```
Comment (cfzjywxk) [inline best-practices/massive-regions-best-practices.md:137]: 是否需要提：
1.  从 v8.3 版本开始，region size 默认大小为 256 MiB 
2. 对新建集群生效，老集群不影响
下同。

@qiancai 
这个修改 PR v8.3 版本已经携带，v8.3 release note 也需补充相关说明。

Comment (qiancai) [reply best-practices/massive-regions-best-practices.md:137]: 是的，正在讨论中

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1081,7 +1081,7 @@ Coprocessor 相关的配置项。
 ### `region-split-size`
 
 + 分裂后新 Region 的大小，此值属于估算值。
-+ 默认值：96MiB
++ 默认值：256MiB
```
Comment (qiancai) [inline tikv-configuration-file.md:1084]:
```suggestion
+ 默认值：256MiB。在 v8.3.0 之前，默认值为 `96MiB`。
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1092,7 +1092,7 @@ Coprocessor 相关的配置项。
 ### `region-split-keys`
 
 + 分裂后新 Region 的 key 的个数，此值属于估算值。
-+ 默认值：960000
++ 默认值：2560000
```
Comment (qiancai) [inline tikv-configuration-file.md:1095]:
```suggestion
+ 默认值：2560000。在 v8.3.0 之前，默认值为 `960000`。
```

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -2150,7 +2150,7 @@ Raft Engine 相关的配置项。
 
 + 备份 SST 文件大小的阈值。如果 TiKV Region 中备份文件的大小超过该阈值，则将该文件备份到 Region 分割的多个 Region 文件中，每个分割 Region 中的文件大小均为 `sst-max-size`（或略大）。
 + 例如，当 Region `[a,e)` 中备份文件大小超过 `sst-max-size` 时，该文件会被备份到多个 Region 范围中，分别为 Region `[a,b)`、`[b,c)`、`[c,d)` 和 `[d,e)`，并且 `[a,b)`、`[b,c)` 和 `[c,d)` 的大小均为 `sst-max-size`（或略大）。
-+ 默认值：`"144MiB"`
++ 默认值：`"384MiB"`
```
Comment (qiancai) [inline tikv-configuration-file.md:2153]:
```suggestion
+ 默认值：`"384MiB"`。在 v8.3.0 之前，默认值为 `144MiB`。
```

File: pd-configuration-file.md [hunk 5]
```diff
@@ -256,13 +256,13 @@ pd-server 相关配置项。
 ### `max-merge-region-size`
 
 + 控制 Region Merge 的 size 上限，当 Region Size 大于指定值时 PD 不会将其与相邻的 Region 合并。
-+ 默认：20
++ 默认：54（8.2.0 及之前版本默认值为 20）
```
Comment (qiancai) [inline pd-configuration-file.md:259]:
```suggestion
+ 默认：54。在 v8.3.0 之前，默认值为 20。
```

File: pd-configuration-file.md [hunk 6]
```diff
@@ -256,13 +256,13 @@ pd-server 相关配置项。
...
 + 单位：MiB
 
 ### `max-merge-region-keys`
 
 + 控制 Region Merge 的 key 上限，当 Region key 大于指定值时 PD 不会将其与相邻的 Region 合并。
-+ 默认：200000
++ 默认：540000（8.2.0 及之前版本默认值为 200000）
```
Comment (qiancai) [inline pd-configuration-file.md:265]:
```suggestion
+ 默认：540000。在 v8.3.0 之前，默认值为 200000。
```

**Comment by LykxSassinator** [comment]
Hold for v8.4. /cc @qiancai

File: best-practices/massive-regions-best-practices.md [hunk 7]
```diff
@@ -134,7 +134,11 @@ config set merge-schedule-limit 8
...
+
+> **注意：**
+>
+> 从 v8.4.0 开始，Region 的默认值从 96 MiB 调整为 256 MiB。
```
Comment (songrijie) [inline best-practices/massive-regions-best-practices.md:141]: For deployments upgraded from a prior version, is there anything to be aware of?

Comment (hfxsd) [inline best-practices/massive-regions-best-practices.md:141]:
```suggestion
> 从 v8.4.0 开始，Region 的默认值从 96 MiB 提高到 256 MiB。如果你并未主动调整过 Region 大小，将集群从 v8.4.0 之前升级至 v8.4.0 或更高的版本后，TiKV 集群的 Region 的默认值将自动更新为 256 MiB。
```

**Review by hfxsd — APPROVED** [review]

**Review by songrijie — APPROVED** [review]

File: pd-configuration-file.md [hunk 8]
```diff
@@ -256,13 +256,13 @@ pd-server 相关配置项。
 ### `max-merge-region-size`
 
 + 控制 Region Merge 的 size 上限，当 Region Size 大于指定值时 PD 不会将其与相邻的 Region 合并。
-+ 默认：20
++ 默认：54。在 v8.4.0 之前，默认值为 20。
```
Comment (lilin90) [inline pd-configuration-file.md:259]:
```suggestion
+ 默认：54。在 v8.4.0 之前，默认值为 20；从 v8.4.0 开始，默认值为 54。
```

File: pd-configuration-file.md [hunk 9]
```diff
@@ -256,13 +256,13 @@ pd-server 相关配置项。
...
 + 单位：MiB
 
 ### `max-merge-region-keys`
 
 + 控制 Region Merge 的 key 上限，当 Region key 大于指定值时 PD 不会将其与相邻的 Region 合并。
-+ 默认：200000
++ 默认：540000。在 v8.4.0 之前，默认值为 200000。
```
Comment (lilin90) [inline pd-configuration-file.md:265]:
```suggestion
+ 默认：540000。在 v8.4.0 之前，默认值为 200000；从 v8.4.0 开始，默认值为 540000。
```

File: tikv-configuration-file.md [hunk 10]
```diff
@@ -1092,7 +1092,7 @@ Coprocessor 相关的配置项。
 ### `region-split-keys`
 
 + 分裂后新 Region 的 key 的个数，此值属于估算值。
-+ 默认值：960000
++ 默认值：`"2560000"`。在 v8.4.0 之前，默认值为 `"960000"`。
```
Comment (lilin90) [inline tikv-configuration-file.md:1095]:
```suggestion
+ 默认值：`2560000`。在 v8.4.0 之前，默认值为 `960000`。
```

File: tune-tikv-memory-performance.md [hunk 11]
```diff
@@ -111,8 +111,8 @@ region-split-check-diff = "32MB"
 
 ## 当区间为 [a,e) 的 Region 的大小超过 `region_max_size`，TiKV 会尝试分裂该 Region，例如分裂成 [a,b)、[b,c)、[c,d)、[d,e) 等区间的 Region 后
 ## 这些 Region [a,b), [b,c), [c,d) 的大小为 `region_split_size` (或者稍大于 `region_split_size`）
-# region-max-size = "144MB"
-# region-split-size = "96MB"
+# region-max-size = "384MB"
+# region-split-size = "256MB"
```
Comment (lilin90) [inline tune-tikv-memory-performance.md:115]: 这里跟 tikv-configuration-file.md 文件里的写法不一致，请确认下是否改为 `"256MiB"`？
```suggestion
# region-max-size = "384MiB"
# region-split-size = "256MiB"
```

Comment (lilin90) [reply tune-tikv-memory-performance.md]: Done in https://github.com/pingcap/docs-cn/pull/18336/commits/01ad78d2a4471444704a69d4314c4d0e99532691

**Review by lilin90 — APPROVED** [review]
/lgtm

File: tune-tikv-memory-performance.md [hunk 12]
```diff
@@ -1,7 +1,7 @@
 ---
 title: TiKV 内存参数性能调优
 aliases: ['/docs-cn/dev/tune-tikv-memory-performance/','/docs-cn/dev/reference/performance/tune-tikv/', '/docs-cn/dev/tune-tikv-performance/']
-summary: TiKV 内存参数性能调优，根据机器配置情况调整参数以达到最佳性能。TiKV 使用 RocksDB 作为持久化存储，配置项包括 block-cache 大小和 write-buffer 大小。除此之外，系统内存还会被用于 page cache 和处理大查询时的数据结构生成。推荐将 TiKV 部署在 CPU 核数不低于 8 或内存不低于 32GB 的机器上，对写入吞吐要求高时使用吞吐能力较好的磁盘，对读写延迟要求高时使用 IOPS 较高的 SSD 盘。
+summary: TiKV 内存参数性能调优，根据机器配置情况调整参数以达到最佳性能。TiKV 使用 RocksDB 作为持久化存储，配置项包括 block-cache 大小和 write-buffer 大小。除此之外，系统内存还会被用于 page cache 和处理大查询时的数据结构生成。推荐将 TiKV 部署在 CPU 核数不低于 8 或内存不低于 32GiB的机器上，对写入吞吐要求高时使用吞吐能力较好的磁盘，对读写延迟要求高时使用 IOPS 较高的 SSD 盘。
```
Comment (hfxsd) [inline tune-tikv-memory-performance.md:4]:
```suggestion
summary: TiKV 内存参数性能调优，根据机器配置情况调整参数以达到最佳性能。TiKV 使用 RocksDB 作为持久化存储，配置项包括 block-cache 大小和 write-buffer 大小。除此之外，系统内存还会被用于 page cache 和处理大查询时的数据结构生成。推荐将 TiKV 部署在 CPU 核数不低于 8 或内存不低于 32GiB 的机器上，对写入吞吐要求高时使用吞吐能力较好的磁盘，对读写延迟要求高时使用 IOPS 较高的 SSD 盘。
```

File: tune-region-performance.md [hunk 13]
```diff
@@ -11,7 +11,7 @@ summary: 了解如何通过调整 Region 大小等方法对 Region 进行性能
 
 TiKV 自动将底层数据进行[分片](/best-practices/tidb-best-practices.md#数据分片)，所有数据按照 key 的范围划分为若干个 Region。当某个 Region 的大小超过一定限制后，TiKV 会将它分裂为多个 Region。
 
-在大量数据的场景下，如果 Region 较小，可能会出现 Region 数量过多的情况，从而带来更多的资源开销和导致[性能回退](/best-practices/massive-regions-best-practices.md#性能问题)的问题。从 v6.1.0 开始，TiDB 支持设置自定义的 Region 大小。Region 默认的大小约为 96 MiB，将其调大可以减少 Region 个数。
+在大量数据的场景下，如果 Region 较小，可能会出现 Region 数量过多的情况，从而带来更多的资源开销和导致[性能回退](/best-practices/massive-regions-best-practices.md#性能问题)的问题。从 v6.1.0 开始，TiDB 支持设置自定义的 Region 大小，该特性在 v6.5.0 正式发布；同时，从 v8.4.0 开始，Region 默认的大小从 96 MiB 调整为 256 MiB，将其调大可以减少 Region 个数。
```
Comment (hfxsd) [inline tune-region-performance.md:14]:
```suggestion
在大量数据的场景下，如果 Region 较小，可能会出现 Region 数量过多的情况，从而带来更多的资源开销和导致[性能回退](/best-practices/massive-regions-best-practices.md#性能问题)的问题。

> **说明：**
>
> - 在 v6.1.0 中，TiDB 支持设置自定义的 Region 大小，该特性为实验特性。
> - 该特性在 v6.5.0 正式发布（GA）。
> - 从 v8.4.0 开始，Region 默认的大小从 96 MiB 调整为 256 MiB，将其调大可以减少 Region 个数。
```

File: tune-region-performance.md [hunk 14]
```diff
@@ -11,7 +11,13 @@ summary: 了解如何通过调整 Region 大小等方法对 Region 进行性能
 
 TiKV 自动将底层数据进行[分片](/best-practices/tidb-best-practices.md#数据分片)，所有数据按照 key 的范围划分为若干个 Region。当某个 Region 的大小超过一定限制后，TiKV 会将它分裂为多个 Region。
 
-在大量数据的场景下，如果 Region 较小，可能会出现 Region 数量过多的情况，从而带来更多的资源开销和导致[性能回退](/best-practices/massive-regions-best-practices.md#性能问题)的问题。从 v6.1.0 开始，TiDB 支持设置自定义的 Region 大小。Region 默认的大小约为 96 MiB，将其调大可以减少 Region 个数。
+在大量数据的场景下，如果 Region 较小，可能会出现 Region 数量过多的情况，从而带来更多的资源开销和导致[性能回退](/best-practices/massive-regions-best-practices.md#性能问题)的问题。
+
+> **说明：**
+>
+> - 在 v6.1.0 中，TiDB 支持设置自定义的 Region 大小，该特性为实验特性。
+> - 该特性在 v6.5.0 正式发布（GA）。
```
Comment (hfxsd) [inline tune-region-performance.md:19]:
```suggestion
> - 从 v6.5.0 开始，该特性成为正式功能（GA）。
```

**Pull request merged**